apps/etterlatte-vilkaarsvurdering/src/main/resources/db/migration/V27__alle_tilganger_cloudbruker.sql

```diff
@@ -0,0 +1 @@
+GRANT ALL PRIVILEGES on all tables in schema vilkaarsvurdering to cloudsqliamuser;
```

Comment:
Litt usikker på hva som er default for `cloudsqliamuser`, tror det er kun `select´. ref https://docs.nais.io/persistence/postgres/how-to/personal-access/?h=privs#prerequisites
Men det gjelder kun for public schemas.